usersync/syncersbuilder.go

```diff
@@ -123,3 +125,37 @@ func chooseSyncerConfig(biddersSyncerConfig []namedSyncerConfig) (namedSyncerCon
 
 	return syncerConfig, nil
 }
+
+func getSyncerKey(biddersWithSyncerCfg map[string]config.BidderInfo, bidderName string, bidderInfo config.BidderInfo) (string, error) {
+	if bidderInfo.Syncer == nil {
+		return "", fmt.Errorf("found no syncer config for alias bidder %s", bidderName)
+	}
```

Comment:
There seems to be a problem here - 
- There is no check before this if statement that input to this function is an alias. The error specifically says `found no syncer config for alias bidder`.
- I think this would also error out in case there are no syncer defined for a bidder. I think that case should not error out.

Reply:
Error message was incorrect. It shouldn't include `alias` in message.  https://github.com/prebid/prebid-server/pull/3082/commits/56da6aa11507faa3144c48ef70b023f0bfa0a864  fixes error message.